proposals/market_perp_btcusd.go

```diff
@@ -131,40 +131,45 @@ func NewBTCUSDPerpetualMarketProposal(
 							"base:BTC",
 							"quote:USD",
 							"class:fx/crypto",
-							"monthly",
+							"perpetual",
 							"sector:crypto",
 						}, extraMetadata...),
 						PriceMonitoringParameters: &vega.PriceMonitoringParameters{
 							Triggers: []*vega.PriceMonitoringTrigger{
 								{
 									Horizon:          43200,
-									Probability:      "0.9999999",
-									AuctionExtension: 600,
+									Probability:      "0.9999",
+									AuctionExtension: 300,
+								},
+								{
+									Horizon:          14400,
+									Probability:      "0.9999",
+									AuctionExtension: 180,
 								},
 								{
-									Horizon:          300,
+									Horizon:          3600,
 									Probability:      "0.9999",
-									AuctionExtension: 60,
+									AuctionExtension: 120,
 								},
 							},
 						},
-						LpPriceRange: "0.5",
+						LpPriceRange: "0.05",
```

Comment:
Shouldn't this be in the SLAParameter section?

Reply:
No idea, just modifying the existing setup, but it compiles so either it's using older protos or it is like that.